packages/editor-ui/src/components/CopyInput.vue

```diff
@@ -2,11 +2,11 @@
 	<div>
 		<n8n-input-label :label="label">
 			<div :class="$style.copyText" @click="copy">
-				<span>{{ copyContent }}</span>
-				<div :class="$style.copyButton"><span>{{ copyButtonText }}</span></div>
+				<span>{{ value }}</span>
+				<div :class="$style.copyButton"><span>{{ copyButtonText || $locale.baseText('generic.copyToClipboard') }}</span></div>
```

Comment:
I think you might be able to set this as default fn for prop instead

Reply:
Good idea. Done!